### PR TITLE
[1.0.1] Workaround to fix UI Update

### PR DESCRIPTION
**Pull request by Madserker**
Stencil cant detect any change done in the properties of a complex object passed by Prop(). So we must specify the variables we want to update te UI outside the field property. In this case: readOnly and required were added as a new inputs in OCR, object counter and email box.

### CODE REVIEW

**Review by SergiDiazCastro — DISMISSED** [review]

File: src/components/email-box/email-box.tsx [hunk 1]
```diff
@@ -14,17 +14,20 @@ export class EmailBox {
...
   @Prop()
-  field!: Field;
+  public field!: Field;
 
   @Prop()
-  control!: any;
+  public control!: any;
+
+  @Prop()
+  public readOnly: boolean;
```
Comment (Xelit3) [inline src/components/email-box/email-box.tsx:23]: please add a TODO comment explaining the workaround for both properties. Future dev's can be confused of why we have separate props when field has everything internally, and we can find anyway in the future to deal with differently.

File: src/components/object-counter/object-counter.tsx [hunk 2]
```diff
@@ -23,6 +23,12 @@ export class ObjectCounter {
   @Prop()
   public control!: any;
 
+  @Prop()
+  public readOnly: boolean;
```
Comment (Xelit3) [inline src/components/object-counter/object-counter.tsx:27]: same here than previous comment

**Review by ericHurtadoAb — APPROVED** [review]

File: src/components/ocr/ocr-field.tsx [hunk 3]
```diff
@@ -18,6 +18,12 @@ export class OcrField {
   @Prop()
   public field!: Field;
 
+  @Prop()
+  public readOnly: boolean;
```
Comment (Xelit3) [inline src/components/ocr/ocr-field.tsx:22]: same here than previous comments

**Review by Xelit3 — APPROVED** [review]
Hi @Madserker !! Approved with a few comments to review easy to take

File: src/components/email-box/email-box.tsx (hunk 1 above)
Comment (Madserker) [reply src/components/email-box/email-box.tsx:28]: Done :)

File: src/components/object-counter/object-counter.tsx (hunk 2 above)
Comment (Madserker) [reply src/components/object-counter/object-counter.tsx:32]: Done :)

File: src/components/ocr/ocr-field.tsx (hunk 3 above)
Comment (Madserker) [reply src/components/ocr/ocr-field.tsx]: Done!

**Review by SergiDiazCastro — APPROVED** [review]

**Review by Xelit3 — APPROVED** [review]

**Review by Xelit3 — APPROVED** [review]

**Review by SergiDiazCastro — APPROVED** [review]

**Pull request merged**